app/models/delivery_partner.rb

```diff
@@ -4,6 +4,8 @@ class DeliveryPartner < DiscardableRecord
   has_many :provider_relationships
   has_many :lead_providers, through: :provider_relationships
 
+  has_paper_trail
```

Comment:
I wonder if we want to form a convention that paper trail is the very first thing in the model, if it has it - so we don't spend time looking for it. Thoughts?

Reply:
No strong opinions either way. I'm sure there will be other things we add that we may want to come first and then soon or later we would have to make a choice and I see that convention breaking subsequently.

Reply:
Yeah, I guess that convention might evolve to "things from external gems come first, then things from Rails, then whatever custom methods we wrote", or something to that effect